fix: hide schedules that are live from /games

samdharris opened the pull request:
### Overview

Resolves #67 by filtering the active games list, removing any games that are currently in the active state. This will stop poorly rendered games from showing up since not enough information is provided to display anything meaningful.

Commit:
fix: hide schedules that are live from /games

stephen-slm commented on pages/games.vue:
```diff
@@ -70,8 +70,13 @@ export default {
                 this.season = Number(newSeason) || 3;
 
-                this.games = await Http.for('games/season').get(this.season);
+                const games = await Http.for('games/season').get(this.season);
```
@samdharris Do you think you could change this request to use the axios object instead of the HTTP? We are moving away from the HTTP objective as and when we work on a area, just switch it over?

You should be able to find other examples in the project, something like `this.$axios.get(url)`?

samdharris replied:
@MambaDev job done

Commit:
fix - swap to axios rather than Http class

fix: hide schedules that are live from /games

amend - swap to axios rather than Http class

Commit:
amend - swap to axios rather than Http class

Commit:
fix - swap to axios rather than Http class

Commit:
Merge branch 'master' of github.com:samdharris/devwars.tv

stephen-slm reviewed (changes requested):
You just need to ensure to correct your commit message styles, this could be done by amending or squashing to a single commit.

Commit:
Merge branch 'master' of github.com:samdharris/devwars.tv
